app/src/test/kotlin/net/rafaeltoledo/social/TestSetup.kt

```diff
@@ -0,0 +1,18 @@
+package net.rafaeltoledo.social
+
+import android.app.Application
+import net.rafaeltoledo.social.di.viewModelModule
+import org.koin.android.ext.android.startKoin
+import org.koin.dsl.module.applicationContext
+
+class TestSocialApp : Application() {
```

Comment:
There is one thing here which worries me a bit: your test application has nothing to do with your production application. If possible, I'd try to minimize this by extending `SocialApp` class instead. This would make initializing Koin a bit more complicated though I think it pays off. IMHO it is really important to have the execution stack as close to the production as we can.